app/app/api/api_v1/endpoints/schools.py

```diff
@@ -0,0 +1,120 @@
+from typing import Any, List
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from app import crud, models, schemas
+from app.api import deps
+
+router = APIRouter()
+
+
+@router.get("/", response_model=List[schemas.School])
+def read_schools(
+    db: Session = Depends(deps.get_db),
+    skip: int = 0,
+    limit: int = 100,
+    current_user: models.User = Depends(deps.get_current_active_superuser),
```

Comment:
For now should we just allow all admins access to these? And update to specific permission when required.

Reply:
I was thinking of doing that once the admin perms are fixed, till then just let it be. Or else it'll just be double the work.

Reply:
Okay